packages/grafana-ui/src/components/Table/DefaultCell.tsx

```diff
@@ -94,7 +94,7 @@ export const DefaultCell = (props: TableCellProps) => {
   );
 };
 
-function getCellStyle(
+export function getCellStyle(
```

Comment:
Is there any reason we'd want to separate this function out from the `DefaultCell` module since it's now being used in a non-DefaultCell way?

Reply:
good call -- removed that and it is much cleaner: https://github.com/grafana/grafana/pull/80691/commits/93ca899fa5e4b5c93a82cf4343ef3b994d4aaeab